features/lwipstack/lwip/src/include/lwip/netif.h

```diff
@@ -136,6 +137,8 @@ enum lwip_internal_netif_client_data_index
 #define NETIF_CHECKSUM_DISABLE_ALL  0x0000
 #endif /* LWIP_CHECKSUM_CTRL_PER_NETIF */
 
+#define INTERFACE_NAME_SIZE    6
```

Comment:
INTERFACE_NAME_SIZE  renamed to  INTERFACE_NAME_MAX_SIZE 
Modified lwipopts.h 
#define INTERFACE_NAME_MAX_SIZE NSAPI_INTERFACE_NAME_MAX_SIZE  to avoid including nsapi_types.h  in lwip   dns and netif